app/src/main/java/com/xa/xpensauditor/TabFragment.java

```diff
@@ -89,28 +108,51 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         arrayAdapterTF=new ArrayAdapter<String>(view.getContext(),android.R.layout.simple_list_item_1,CatgTF);
 
         RefCat.addChildEventListener(new ChildEventListener() {
+            /**
+             * Triggered when a child node is added.
+             * @param dataSnapshot
+             * @param s
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.